static/includes/DisruptiveActionsList.md

```diff
@@ -0,0 +1,27 @@
+&NewLine;
+
+Configuring features and functions can result in disruption to services and access to users or system connecting to TrueNAS through sharing or network services.
+When adding new or changing existing functions and services, plan maintenance windows to minimize disruptions based on the type and area of change.
+
+{{< expand "Actions that Cause Disruptions" "v" >}}
+The following administrator actions  result in disruptions to services and client connections.
+Durtation of the disruptions depends on system and service configurations, number of users accessing shares and services, network interfaces, and configuration methods (directory service provisioned verses manually configured).
+{{< truetable >}}
+| Service | Function | Action Description |
+|---------|----------|-------------|
+| Network |  | Changing any network setting such as adding or modifying an interface, making a general network change, etc., disrupts network services. An improperly configured network interface can result in lost access to the web UI if changing the primary interface address. Improperly configured general network settings such as DNS name servers, default gateway, host name, or domain name can result in broken network connectivity. |
+| Directory Services |  | Changing Active Directory or LDAP services can result in issues with user access, domain access, etc. TrueNAS alerts users when changes cause breakage in affected areas. |
+| SMB service | `smb.update` | Updating all SMB clients disrupts access. Before making changes, the service must be stopped and then restarted when complete. Stopping the service disrupts clients relying on the service. |
+| NFS service | `nfs.update` | Updating all NFS clients disrupts access. Before making changes, the service must be stopped and then restarted when complete. Stopping the service disrupts clients relying on the service. |
+| SMB shares | `sharing.smb.update`<br>`sharing.smb.delete` | Changing `guest` parameter or toggling `home` on or off in all clients disrupts access, otherwise clients connected to a particular share are disrupted. Before making changes, the service must be stopped and then restarted when complete. Stopping the service disrupts clients relying on the service. |
```

Comment:
"in all clients disrupts access, otherwise clients connected to a particular share are disrupted"

I'm having trouble following this

Reply:
Reworded this sentence.